add pilot guide (#141)
Promptless/docs: fern/docs/pilot-guide.mdx

# Your Promptless Pilot

Promptless offers pilot periods for enterprise customers. This pilot period allows you to see how Promptless works with your own documentation, team workflows, and trigger events. The pilot period is 30 days. Below you'll get a sense of what you can expect during your Promptless pilot. 

> ## Why 30 Days?

Reviewer: ```suggestion
```

The 30-day pilot lets you:

- See Promptless handle different types of documentation updates
- Evaluate suggestion quality across different scenarios
- See how well Promptless can integrate into your team's workflow

<Info>
Your pilot starts when Promptless creates your first suggestion—not when you sign up. Therefore, any time that's spent getting approvals for data source connections doesn't count against your pilot.
</Info>

## Before Your Pilot Begins

Before the pilot begins, you'll need to connect Promptless to your relevant workspaces and share resources with the Promptless team so that the Promptless agent can draft an initial batch of suggestions. 

### Setting up Promptless Connectors

Connect Promptless to your key systems:

> - **Source code repositories**: GitHub, GitHub Enterprise, or Bitbucket

Reviewer: ```suggestion
- **Source code repositories**: GitHub, GitHub Enterprise, GitLab, or Bitbucket
```

- **Communication platforms**: Slack or Microsoft Teams  
- **Project management tools**: Jira or Linear (optional, but recommended to give Promptless better context about the customer-facing impact of new features)

See our [Setup & Quickstart guide](/docs/getting-started/setup-quickstart) for integration instructions.

### Set Up Slack Connect

We'll create a shared Slack Connect channel between your team and ours for questions, feedback, and quick support throughout your pilot.

### Share Your Documentation Context

Promptless works best when it has the same sort of information that you'd give to a new technical writer joining your team. 

This can include:

- Your style guide
- Vale configuration (if you use it)
- Documentation standards or templates

## In-Person Onboarding Workshop

> We always attempt to kick off customer pilots with an in-person workshop. 

Reviewer: ```suggestion
We always recommend kicking off customer pilots with an in-person workshop. 
```

### What to Expect

<CardGroup cols={1}>
  <Card title="Duration" icon="fa-regular fa-clock">
    90-minute session. For larger teams, we may do multiple workshops over a few days.
  </Card>
  
  <Card title="Preparation" icon="fa-regular fa-list-check">
    Before your session, we run Promptless on historical events (like the last 30 days of PRs) so you can review real suggestions during the workshop.
  </Card>
  
  <Card title="What We'll Cover" icon="fa-regular fa-graduation-cap">
    How Promptless works, your configuration, and hands-on review of suggestions—with the goal of publishing your first doc updates together.
  </Card>
</CardGroup>

### Workshop Goals

> <Steps>

Reviewer 2: these arent actually steps?

  <Step title="Understand your Promptless setup">

>     Review your configuration and how triggers, context sources, and publishing work for your team

Reviewer 2: i'd want links to pages that define or clarify what each of these terms of art actually means "triggers, context sources"

  </Step>
  <Step title="Publish your first updates">
    Review suggestions together and publish documentation updates during the session
  </Step>
  <Step title="Define success criteria">
    Agree on what a successful pilot with Promptless would look like for your team
  </Step>
</Steps>

> ## Measuring Success for your Pilot

Author: actually disagree strongly. shared success criteria helps reduce surprises when pilots come to a close

> During the workshop, we will want to agree on success criteria for a pilot. This can sometimes change based on each team's individual needs, but based on our experience onboarding dozens of teams, *suggestion volume* and *suggestion quality* are the most reliable success criteria that indicate that your team will have a positive experience with Promptless going forward.

Reviewer 2: this second sentence is kinda brutal
break up. or simplify

### Suggestion Volume

Track how many suggestions you're getting from Promptless. More suggestions isn't necessarily better—sometimes too many suggestions can feel noisy or overwhelming—but a healthy stream of documentation suggestions is a strong indicator that Promptless is plugged into the right parts of your workflow. 

**Typical volume benchmarks:**

> - Teams with 0-1 people working full-time on docs: 20-40 suggestions per month

Reviewer 2: is there a fern component to render this nicely
